modules/platform/nuxeo-elasticsearch/nuxeo-elasticsearch-core/src/test/java/org/nuxeo/elasticsearch/test/TestMapping.java

```diff
@@ -128,8 +128,13 @@ public void testIlikeSearch() throws Exception {
         Assert.assertEquals(0, ret.totalSize());
         ret = ess.query(new NxQueryBuilder(session).nxql("SELECT * FROM Document WHERE dc:description LIKE 'Upper%'"));
         Assert.assertEquals(0, ret.totalSize());
-        ret = ess.query(new NxQueryBuilder(session).nxql("SELECT * FROM Document WHERE dc:description LIKE 'UPPER%'"));
-        Assert.assertEquals(0, ret.totalSize());
+        // match_phrase_prefix query against a keyword field behaves differently depending on the Elasticsearch version
+        // prior to 6.7.0, it doesn't match
+        // since 6.7.0, it matches, yet we're not sure if it is really expected
+        // since 7, it raises an exception
+        // ret = ess.query(new NxQueryBuilder(session).nxql("SELECT * FROM Document WHERE dc:description LIKE
+        // 'UPPER%'"));
+        // Assert.assertEquals(0, ret.totalSize());
```

Comment:
What's the point of keeping commented code/test here? It will be never executed/run.

Reply:
It's to illustrate this interesting mapping edge case and make it easily testable against an ES cluster of the right version.
@bdelbosc WDYT?

Reply:
we don't mind, this is a temporary code until the es7 upgrade is merged